### ISSUE TITLE
[typescript-angular] Broken uploads since updating to 3.0.14

### ISSUE THREAD

**Issue by louisl** [swagger-api/swagger-codegen-generators#553]
I think something broke with this PR https://github.com/swagger-api/swagger-codegen-generators/pull/526, It fixed some issues where I had to manually fix stuff but now I'm getting this,
```typescript
return this.httpClient.post<StorageObject>(`${this.basePath}/storage-objects/${encodeURIComponent(String(storageObjectId))}/upload`,
            {
                withCredentials: this.configuration.withCredentials,
                headers: headers,
                observe: observe,
                reportProgress: reportProgress
            }
        );
```

instead of this,
```typescript
return this.httpClient.post<StorageObject>(`${this.basePath}/storage-objects/${encodeURIComponent(String(storageObjectId))}/upload`,
           // The line below is now missing.
            convertFormParamsToString ? formParams.toString() : formParams, 
            {
                withCredentials: this.configuration.withCredentials,
                headers: headers,
                observe: observe,
                reportProgress: reportProgress
            }
        );
```

**Comment by louisl** [comment]
Also mentioned here https://github.com/swagger-api/swagger-codegen/issues/9890, not sure why he closed it though

**Comment by cmalard** [comment]
Hey @louisl , 3.0.16 is not functionnal (fixed in #576) but can you test it to check if body param is back ?

**Comment by HugoMario** [comment]
tested and confirm this is fixed
